lib/tool_belt/release_environment.rb

```diff
@@ -20,20 +20,53 @@ def setup(args = {})
       Dir.chdir(release_directory) do
         @repos.each do |name, repo|
           @systools.execute("git clone #{repo[:repo]} #{name}") if !File.exist?(name.to_s)
-          if github_username
-            Dir.chdir(name.to_s) do
+
+          Dir.chdir(name.to_s) do
+            @systools.execute("git remote set-url origin #{add_username(repo[:repo], github_username)}") if github_username
```

Comment:
it doesn't point to the users fork, its adding their username to the url for the main fork to keep from having to type it in when pushing